Skip flaky tests with cuSPARSE as its API is quite variable,
might solve itself in future versions of CUDA or jax
see https://github.com/google/jax/issues/14663
author: Alex Domingo (Vrije Universiteit Brussel)
--- tests/sparse_test.py.orig	2023-04-06 23:41:58.192895000 +0200
+++ tests/sparse_test.py	2023-04-06 23:59:56.319161821 +0200
@@ -1173,6 +1173,7 @@
     ],
     dtype=jtu.dtypes.floating + jtu.dtypes.complex,
   )
+  @unittest.skip("CUSPARSE_OPERATION_NON_TRANSPOSE is not supported with CUSPARSE_SPMM_COO_ALG2 on CUDA 11.7.0")
   def test_bcoo_dot_general_cusparse(
     self, lhs_shape, rhs_shape, dtype, lhs_contracting, rhs_contracting):
     rng = jtu.rand_small(self.rng())
